### PR TITLE
Improve setup

### PR DESCRIPTION
**Pull request by jonallured**
While working on setting up energy today, I found some things that were out of date or missing from the getting started point of view. This PR reflects the things that I think might help the next person trying to setup energy. I'd love some feedback! 😄

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -1,43 +1,68 @@
...
+$ make artsy
+```
+
+And if you're setting up as an Open Source contributor, run this:
+
+```
+$ make oss
+```
+
+Set an environment variable of `ARTSY_STAFF_MEMBER` to `true`.
```
Comment (orta) [inline docs/getting_started.md]: this should go above in the being an artsy employee, this originally was in the mobile playbook, but it makes less sense now there is no mobile team really - https://github.com/artsy/mobile/blob/master/playbook.md

File: docs/getting_started.md [hunk 2]
```diff
@@ -1,43 +1,68 @@
...
-Use your `.energy` file
------------------------
+## Use your `.energy` file
 
-Authentication is a lot easier when you don't type so much, create a file in your home directory called `.energy` and it takes a collection of `key:value` lines to have the username and password set for you in the `ARLoginViewController`.  You can use the `ARDeveloperOptions` class to react to the key value store. For example:
+Authentication is a lot easier when you don't type so much, try this:
 
+```
+echo 'user@example.com:shhh' > ~/.energy
+```
```
Comment (orta) [inline docs/getting_started.md]: 👍

File: Artsy Folio.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -3716,12 +3716,12 @@
...
 				606B1EDD184681FF00CC3DBD /* Black.png in Resources */,
 				60B340B71609C28D002A5926 /* garbage_btn_solidwhite.png in Resources */,
-				60C279BD1BE2310300229C8C /* (null) in Resources */,
+				60C279BD1BE2310300229C8C /* BuildFile in Resources */,
 				60B340B91609C28D002A5926 /* garbage_btn_solidwhite@2x.png in Resources */,
 				60B340BB1609C28D002A5926 /* garbage_btn_whiteborder.png in Resources */,
 				60B340BD1609C28D002A5926 /* garbage_btn_whiteborder@2x.png in Resources */,
 				606B1EDB1846805E00CC3DBD /* Transparent.png in Resources */,
-				60C279BC1BE2310300229C8C /* (null) in Resources */,
+				60C279BC1BE2310300229C8C /* BuildFile in Resources */,
```
Comment (orta) [inline Artsy Folio.xcodeproj/project.pbxproj]: what is a build file?

**Comment by orta** [comment]
@jonallured prefer using "assign" over "reviewer" as it works with our slack bot 

https://github.com/alloy/no-slacking-on-pull-requests-bot

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: Artsy Folio.xcodeproj/project.pbxproj (hunk 3 above)
Comment (jonallured) [inline Artsy Folio.xcodeproj/project.pbxproj]: lol, I don't know! I don't know why that showed up there!

**Comment by orta** [comment]
> as it works with our slack bot

( mainly because there's no API to get reviewers yet )

I'm fine with the Danger fail on merging this too 👍

**Pull request merged**